servers/rendering/renderer_rd/renderer_scene_render_forward.h

```diff
@@ -46,7 +47,8 @@ class RendererSceneRenderForward : public RendererSceneRenderRD {
 
 	enum {
 		SDFGI_MAX_CASCADES = 8,
-		MAX_GI_PROBES = 8
+		MAX_GI_PROBES = 8,
+		MAX_LIGHTMAPS = 8
```

Comment:
Probably best to keep a trailing comma:

```suggestion
		MAX_LIGHTMAPS = 8,
```